test/mainnet/ynbtck.spec.sol

```diff
@@ -442,6 +446,10 @@ contract YnBTCkTest is Test, AssertUtils, MainnetKernelActors, EtchUtils, VaultU
     }
 
     function test_Vault_ynBTCk_deposit_EnzoBTC(uint256 btcbAmount, bool alwaysComputeTotalAssets) public {
+        // skip test if ENZO_NETWORK is paused
+        if (IPausable(ENZO_NETWORK).paused()) {
```

Comment:
Enzo tests were failing since it looks like they have paused their contracts, I added this check.
Let me know if this is okay or if we should let the tests fail for the time being.